content/rs/release-notes/rs-7-2-releases/rs-7-2-tba.md

```diff
@@ -28,11 +48,293 @@ The following table shows the MD5 checksums for the available packages:
 ## New features and enhancements
 
-TBA
+#### Redis 7.0 features
+
+The following Redis 7.0 features are now supported:
+
+- [Redis functions](https://redis.io/docs/interact/programmability/functions-intro/)
+
+    In Redis Enterprise Software, [`FUNCTION STATS`](https://redis.io/commands/function-stats/) returns an extra parameter, an array called `all_running_scripts`, to reflect multiple functions running at the same time.
+
+- [Multipart AOF](https://redis.io/docs/management/persistence/#append-only-file) (append-only files)
+
+- New commands
+
+- Sharded `PUBSUB` (see [Sharded pub/sub](#sharded-pubsub) for details)
+
+#### Redis 7.2 features
+
+The following Redis 7.2 features are now supported:
+
+- Various performance improvements
+
+- `CONFIG SET` for locale
+
+- Module API improvements
+
+- Connection layer modularization
+
+- Encoding improvements: listpack for sets and lists
+
+- Observability: authentication metrics (exposed by `INFO` command)
+
+- Stream consumer group improvements
+
+- Commands: `ZRANK`, `ZREVRANK` new `WITHSCORE` option
+
+- Shard IDs in cluster shards topology
+
+- Introduce shard ID to Redis cluster
+
+- Support `CLIENT NO-TOUCH` command
+
+- `WAIT AOF`
+
+#### Three Redis database versions
+
+Redis Enterprise Software version 6.x includes two Redis database versions: 6.0 and 6.2. As of version 7.2, Redis Enterprise Software includes three Redis database versions: 6.0, 6.2, and 7.2.
+
+To view available Redis database versions:
+
+- In the Cluster Manager UI, see **Redis database versions** on the **Cluster > Configuration** screen.
+
+- Send a [`GET /nodes` REST API request]({{<relref "/rs/references/rest-api/requests/nodes">}}) and see `supported_database_versions` in the response.
+
+The default Redis database version, which is used when you upgrade an existing database or create a new one, differs between Redis Enterprise releases as follows:
+
+| Redis<br />Enterprise | Bundled Redis<br />DB versions | Default DB version<br />(upgraded/new databases) |
+|-------|----------|-----|
+| 7.2 | 6.0, 6.2, 7.2 | 7.2 |
+| 6.4.2 | 6.0, 6.2 | 6.2 |
+| 6.2.x | 6.0, 6.2 | 6.0 |
+
+For Redis Enterprise Software version 7.2, `default_redis_version` is 7.2 for both `major` and `latest` upgrade policies.
+
+#### Auto Tiering - Redis on Flash evolution doubling performance {#auto-tiering}
+
+Redis Enterprise version 7.2 introduces Auto Tiering as an enhanced successor to Redis on Flash, which allows you to provision larger databases at a lower cost by using NVMe drives in addition to RAM.
+
+Redis Enterprise Auto Tiering uses [Speedb](https://www.speedb.io/) as its storage engine, doubling the throughput achieved. For example, a 1 TB database with 50K ops/sec can now serve 100K ops/sec based on the same infrastructure.
+
+For more information about Auto Tiering, see:
+
+- [Auto Tiering overview]({{<relref "/rs/databases/auto-tiering">}})
+
+- [Auto Tiering quick start]({{<relref "/rs/databases/auto-tiering/quickstart">}})
+
```

Comment:
I would like to include here another sentence or paragraph that explains what customers need to do in order to utilize Auto Tiering with Speedb:
Customers will need to upgrade their cluster to RS 7.2, and then upgrade each of their Auto Tiering enabled databases to v7.2 (DB version) so it will start using Speedb and get the performance improvements.

Reply:
Added this info